packages/website/src/components/lib/markdown.ts

```diff
@@ -29,25 +29,51 @@ function createSummaryJson(
   return '';
 }
 
+export function genVersions(state: ConfigModel): string {
+  return [
+    '| package | version |',
+    '| -- | -- |',
+    `| \`@typescript-eslint/eslint-plugin\` | \`${process.env.TS_ESLINT_VERSION}\` |`,
+    `| \`@typescript-eslint/parser\` | \`${process.env.TS_ESLINT_VERSION}\` |`,
+    `| \`TypeScript\` | \`${state.ts}\` |`,
+    `| \`ESLint\` | \`${process.env.ESLINT_VERSION}\` |`,
+    `| \`node\` | \`web\` |`,
+  ].join('\n');
+}
+
 export function createMarkdown(state: ConfigModel): string {
   return [
-    '## Repro',
     `[Playground](${document.location.toString()})`,
     createSummary(state.code, 'Code', 'ts', 30),
     createSummaryJson(state.rules, 'rules', 'Eslint config'),
     createSummaryJson(state.tsConfig, 'compilerOptions', 'TypeScript config'),
-    '## Expected Result\n',
-    '## Actual Result\n',
-    '## Additional Info\n',
-    '## Versions',
-    `| package                            | version |
-| ---------------------------------- | ------- |
-| \`@typescript-eslint/eslint-plugin\` | \`${process.env.TS_ESLINT_VERSION}\` |
-| \`@typescript-eslint/parser\`        | \`${process.env.TS_ESLINT_VERSION}\` |
-| \`TypeScript\`                       | \`${state.ts}\` |
-| \`ESLint\`                           | \`${process.env.ESLINT_VERSION}\` |
-| \`Env\`                              | \`web\` |`,
+    genVersions(state),
   ]
     .filter(Boolean)
     .join('\n\n');
 }
+
+export function createMarkdownParams(state: ConfigModel): string {
+  const params = {
+    labels: 'bug,package:+eslint-plugin,triage',
+    template: '1-bug-report-plugin.yaml',
+    title: 'Bug:+[rule+name+here]+<short+description+of+the+issue>',
```

Comment:
> You can wrap this in encodeURI and it'll do it for you

not needed, encoding is handled by `URLSearchParams`, all what i had to correct is replace `+` with ` `

Reply:
I believe `URLSearchParams` will convert spaces to `+`

```
new URLSearchParams({ foo: 'a b c' }).toString()
'foo=a+b+c'
```

Reply:
Oh I see - because you manually wrote `+`, it encoded the `+` in the param, instead of leaving it as a `+`.
Otherwise it will encode a space as a raw `+` and GH will decode it properly